tests/conftest.py

```diff
@@ -860,41 +910,53 @@ def sample_config(tumor_sample_name: str, normal_sample_name: str):
 
 @pytest.fixture(scope="session")
 def analysis_path():
-    """Analysis test path"""
+    """
+    Return path for test analysis
+    """
     return "tests/test_data/qc_files/analysis"
 
 
 @pytest.fixture(scope="session")
 def multiqc_data_path(analysis_path):
-    """multiqc_data.json test path"""
+    """
+    Return path of JSON for MultiQC test data
+    """
```

Comment:
The idea is to have one-line docstrings where they fit. We need to refactor the existing ones, not the new ones that have the correct format.

```suggestion
    """Return path of JSON for MultiQC test data."""
```

Reply:
Continuing from the reply about the consistency above, would it not be nice to keep the format consistent across the code, which will make the format easy to follow?

Reply:
Yes, and the way to do it is following the coding guidelines: https://peps.python.org/pep-0257/#one-line-docstrings